src/drivers/mediatek/mt8195/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 add_local_sources(sof ipc.c timer.c
-	interrupt.c uart.c afe-memif.c afe-dai.c afe-drv.c
+	interrupt.c afe-memif.c afe-dai.c afe-drv.c
 )
 
```

Comment:
So, this was added by mistake. Please create a separate patch which removes the files added to build system by mistake.